tools-local/tasks/installer.tasks/installer.tasks.csproj

```diff
@@ -16,8 +13,8 @@
 
   <ItemGroup>
     <PackageReference Include="NuGet.ProjectModel" Version="$(RefOnlyNugetProjectModelPackageVersion)" />
-    <PackageReference Include="Microsoft.Extensions.DependencyModel" Version="2.1.0" />
-    <PackageReference Include="Microsoft.DotNet.PlatformAbstractions" Version="2.1.0" />
+    <PackageReference Include="Microsoft.Extensions.DependencyModel" Version="$(MicrosoftExtensionsDependencyModelPackageVersion)" />
+    <PackageReference Include="Microsoft.DotNet.PlatformAbstractions" Version="$(MicrosoftDotNetPlatformAbstractionsPackageVersion)" />
```

Comment:
This changes the version from 2.1.0 to 3.0.0, which impacts source-build. Is it necessary in this PR?

Reply:
Yes, I was also concerned about this change. I will replace it back to the 2.1.0 version but we should consolidate the different versions soon.